README.md

```diff
@@ -175,20 +175,20 @@ Components:
 * core: rolls up all of the rest-assured groovy syntax into simple RestApiController.\<method> calls (i.e. RestApiController.put(req, resp) and handles the firing of the call based on the config-util input.
 * fuzzer-lib: exhaustively fuzzes a REST body with negative, edge-case, and malicious (though safe for testing) calls to the service.
-* performance-lib: Scala libary based on Gatling which allows you to place your scala-modified test code in snippet block and run performance metrics
+* performance-lib: Scala library based on Gatling which allows you to place your scala-modified test code in snippet block and run performance metrics
 
 ### Building and Release Versions
 Release versions are created as PRs are reviewed and made widely available. 
 
 
 ### Core Extensions -
-When a service REST API comes online, in order to test it with Adobe Ride, the Core will need to be extended to a service-library.  This is done in order to handle workflows and data specific to that API/Service.
+When a service REST API comes online, to test it with Adobe Ride, the Core will need to be extended to a service-library.  This is done to handle workflows and data specific to that API/Service.
 
 ### SharedResources
-Shared resources is a term used to describe a set of resources you create for your Ride extension project (just like any other resource in a javaproject), but specifically json schema for datamodeling and generation, and config files for use in endpoint targeting and testuser management. Ideally your shared-resources resources lives in separate and centralized source repo outside of your extension extension project, built as a jar, and then used as a dependency for your project (explained in the useage docs). This is so that if, as is common, you are working in an environment creating multiple webservices/API, there can be one set of resources that can be used by multiple extensions for easy end to end testing.  However, it is just as valid to put these resources in the resources directory of your java project (src/main/resources).
+Shared resource is a term used to describe a set of resources you create for your Ride extension project (just like any other resource in a javaproject), but specifically json schema for data modeling and generation, and config files for use in endpoint targeting and testuser management. Ideally, your shared-resources resources lives in separate and centralized source repo outside of your extension project, built as a jar, and then used as a dependency for your project (explained in the usage docs). This is so that if, as is common, you are working in an environment creating multiple webservices/API, there can be one set of resources that can be used by multiple extensions for an easy end to end testing.  However, it is just as valid to put these resources in the resources directory of your java project (src/main/resources).
```

Comment:
good catch.  Would you mind also changing "...lives in separate..." to "...lives in **a** separate..."

Reply:
Okk did it.